source/install/transport-encryption/config.rst

```diff
@@ -0,0 +1,22 @@
+=============================================
+Configuring Transport Encryption
+=============================================
+
+The components of the Mattermost setup are shown in the following diagram,
+including the transport encryption used. Aside from the encryption between the
+Mattermost Application Server, all transports rely on TLS encryption.
```

Comment:
between the Mattermost Application Server and *what*?

Reply:
Guessing this should be `Mattermost Application Servers` but I'm certain?

Reply:
Edited to `Aside from the encryption between the nodes
of the Mattermost cluster, all transports rely on TLS encryption.`, which should make that more clear.